openshift-ci/wait-for-csv.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+VERSION="v0.0.0"
+CSV_NAME="metallb-operator.${VERSION}"
+NAMESPACE="metallb-system"
+
+ATTEMPTS=0
+MAX_ATTEMPTS=60
+csv_created=false
+until $csv_created || [ $ATTEMPTS -eq $MAX_ATTEMPTS ]
+do
+    echo "waiting for csv to be created attempt:${ATTEMPTS}"
+    if oc get csv -n $NAMESPACE $CSV_NAME; then
+        echo "csv created!"
+        csv_created=true
+    else    
+        echo "failed, retrying"
+        sleep 5
+    fi
+    (( ATTEMPTS++ ))
+done
+
+if ! $csv_created; then 
+    echo "Timed out waiting for csv to be created"
```

Comment:
This is the same logic we have in the metallb-operator, I took it as it is assuming it was done in that way for a reason. Also, it won't worth knowing if the csv is not created or it's failing because that will show different issues.